src/serde_helpers.rs

```diff
@@ -82,6 +84,14 @@ pub fn serialize_u64_as_i64<S: Serializer>(val: &u64, serializer: S) -> Result<S
     }
 }
 
+/// Serializes an [`ObjectId`] as a hex string.
+pub fn serialize_object_id_as_hex_string<S: Serializer>(
+    val: &ObjectId,
+    serializer: S,
+) -> Result<S::Ok, S::Error> {
+    val.to_string().serialize(serializer)
```

Comment:
Can we use `to_hex` here instead? just in case we ever change the `Display` implementation.